react/components/FilterOptionTemplate.js

```diff
@@ -88,6 +88,8 @@ const FilterOptionTemplate = ({
   navigateToFacet,
   showClearByFilter,
   preventRouteChange,
+  handleClear,
+  showClearButton,
```

Comment:
will it only be used by PriceRange? if so, why don't you use the `showClearByFilter` prop itself? 
since PriceRange doesn't have `filters`, you could check if `showClearByFilter` is true && `id` is 'priceRange'

Reply:
Good catch

you mean like this right?

```js
const isClearButtonVisible =
    showClearByFilter &&
    (id === 'priceRange' ||
      (!selected && filters && filters.some(filter => filter.selected)))
```

Done

https://clearall2--storecomponents.myvtex.com/top?_q=top&map=ft